site-overlay/layout.erb

```diff
@@ -87,9 +105,9 @@
         <div class="col-sm text-center">
           <a target="_blank" href="https://github.com/siteadmin/inferno/issues">Issues</a>
         </div>
-        <div class="col-sm text-right">
-          Version <%= version %>
-        </div>
+        <a href="#" data-toggle="modal" data-target="#ServerStatusModal">
```

Comment:
This is missing a `<div class="col-sm text-right">`

Reply:
Good catch. Updated & force-pushed.